ezgui/src/backend_glow.rs

```diff
@@ -158,10 +100,6 @@ impl<'a> GfxCtxInnards<'a> {
             }
         }
     }
-
-    pub fn finish(self) {
```

Comment:
This method was the only divergence in GfxCtx between wasm vs. native. Like most of the remaining divergence in between native/wasm, it involves window vs. windowed_context.

I did some plumbing so that GfxCtx no longer knows about window vs. windowed_context, allowing wasm and native to have the same GfxCtx implementation.